lease/lease_queue.go

```diff
@@ -16,17 +16,18 @@ package lease
 
 // LeaseWithTime contains lease object with expire information.
 type LeaseWithTime struct {
-	id         LeaseID
-	expiration int64
-	index      int
+	id LeaseID
+	// Unix nanos timestamp.
+	time  int64
```

Comment:
Can we comment this `time` field? It can be either expiration timestamp or checkpoint timestamp. Took me awhile to find out how `time` is being used :)

Reply:
Looks like the field rename from `expiration` to `time` only got me from misleading to unclear. I'll add a comment and see if there is anything else I should do to make this more obvious.

Reply:
Added a couple comments to both `lease_queue.go` and the two places where the time field is used in `lessor.go`.